### ISSUE TITLE
Remove postUrl() special case

### ISSUE THREAD

**Issue by bocops** [andregasser/bigbone#49]
`MastodonClient` currently has both a `post()` and a `postUrl()` method. After #35 is resolved, the latter can be removed and calls to it be replaced with calls to the former, potentially avoiding cases where `postUrl()` is used to post to a URL with a different host than what the client is configured for.

**Comment by andregasser** [comment]
Agree. https://github.com/andregasser/bigbone/issues/35 is now merged into master since couple of minutes.
